packages/server/src/utils/backups/utils.ts

```diff
@@ -256,3 +256,22 @@ export const getBackupCommand = (
 	echo "Backup done ✅" >> ${logPath};
 	`;
 };
+
+export const getRcloneUploadCommand = (
+	destination: Destination,
+	localFilePath: string,
+	remoteFileName: string,
+	prefix: string = ''
+) => {
+	if (destination.type && destination.type !== 's3') {
+		// Use rcloneConfig as the remote
+		const remote = destination.rcloneConfig || '';
+		// e.g. rclone copyto "/tmp/file.zip" "gdrive,client_id=xxx,token=yyy:/backup/file.zip"
+		return `rclone copyto "${localFilePath}" "${remote}${prefix ? '/' + prefix : ''}/${remoteFileName}"`;
```

Comment:
Is it possible to save the config into a file and let rclone read from that file. after upload, also save the data?

Because in some cases rclone updates token using refresh token and saves that back into the file.